rules/prefer-number-properties.js

```diff
@@ -36,10 +36,22 @@ const propertiesSelector = [
 			'MemberExpression[computed=false] > .property',
 			'FunctionDeclaration > .id',
 			'ClassDeclaration > .id',
+			// TODO: remove `ClassProperty` when `babel` and `typescript` support `FieldDefinition`
 			'ClassProperty[computed=false] > .key',
+			'FieldDefinition[computed=false] > .key',
```

Comment:
Oops, I didn't mean to add `FieldDefinition`, but `PropertyDefinition`, I'll fix.